util/dvsim/dvsim.py

```diff
@@ -411,19 +416,19 @@ def parse_args():
 
     buildg.add_argument("--build-opts",
                         "-bo",
-                        nargs="+",
-                        default=[],
-                        metavar="OPT",
+                        default='',
+                        metavar="OPTS",
```

Comment:
The above arg --select_cfgs has a 3 letter singular metavar that hasn't been made plural, should these be consistent?

Reply:
Good point! I'll add an 's' above.